temporalcli/commandsmd/commands.md

```diff
@@ -655,7 +663,107 @@ The temporal task-queue list-partition command displays the partitions of a Task
 
 * `--task-queue`, `-t` (string) - Task queue name. Required.
 
-### temporal task-queue update-build-ids: Operations to update the sets of worker Build ID versions on the Task Queue.
+
+### temporal task-queue update-build-id-rules: Updates the worker Build ID assignment and redirect rules on the Task Queue.
+
+Provides various commands for adding, removing, or replacing worker Build ID assignment and redirect rules associated with a Task Queue. See the help of each sub-command for more.
+
+### temporal task-queue update-build-id-rules add-redirect-rule: Adds the rule to the list of redirect rules for this Task Queue.
+
+Adds a new redirect rule for this Task Queue. There can be at most one redirect rule for each distinct source build ID.
+
+#### Options
+
+* `--source-build-id` (string) - The source build ID for this redirect rule. Required.
+* `--target-build-id` (string) - The target build ID for this redirect rule. Required.
+* `--task-queue`, `-t` (string) - Name of the Task Queue. Required.
+* `--yes`, `-y` (bool) - Skip confirmation.
```

Comment:
Put these option in the parent command if they're shared across all children

Reply:
There is no --yes in get-rules (after I move it under `versioning`), but I'll move task-queue to the parent...